main.py

```diff
@@ -1,45 +1,42 @@
-import requests
 import json
 from flask import Flask, request
-app = Flask(__name__)
-
-REQUIRED_LABELS = ['ready for review']
+from PullRequest import PullRequest
+from config import REQUIRED_LABELS_ALL, REQUIRED_LABELS_ANY, BANNED_LABELS
 
-# config = {
-#     "all": ["label1", "label2"],
-#     "not": ["label3"],
-#     "any": ["label4", "label5"]
-# }
+app = Flask(__name__)
 
 
-@app.route('/', methods=["POST"])
+@app.route('/', methods=["POST", "GET"])
 def main():
-    pull_request = PullRequest(request.get_json())
-    return create_status_json(any(l['name'] in REQUIRED_LABELS for l in pull_request.labels))
+    #pull_request = PullRequest(request.get_json())
+    pull_request = PullRequest()
+    pull_request.set_issue_url('https://api.github.com/repos/dimagi/commcare-android/issues/1858')
```

Comment:
did you mean to leave this here?

Reply:
This was just for testing. I've reworked this a bit in a newer commit, but it's still in a format meant for testing since we don't have it hosted yet